src/app/users/user-edit/user-edit.component.ts

```diff
@@ -24,31 +22,29 @@ export class UserEditComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
+    this.editNewForm = new FormGroup({
```

Comment:
@TesaRocks this is OK.

But I requested you to use the formBuilder approach.

that's is the angular  magic 🪄✨

Please change this one more time mate.

Reply:
Done many thnaks!